mail_tracking/models/mail_message.py

```diff
@@ -99,6 +99,20 @@ def _tracking_status_map_get(self):
             "soft-bounced": "error",
         }
 
+    def _notification_status(self, notification):
```

Comment:
I’ve also added the `@api.model`. It might have been better to put it into a mail_notification.py but it was simpler to see the expected values in the same file.